apps/frontend-pwa/src/pages/editProfile.tsx

```diff
@@ -145,85 +164,179 @@ function EditProfile() {
             },
           })
 
-          if (result.data?.updateParticipantProfile) {
-            router.replace(decodedRedirectPath)
+          if (result.data?.updateParticipantProfile && !result.errors) {
+            // router.replace(decodedRedirectPath)
           } else {
             setShowError(true)
           }
         }}
       >
         {({ values, isSubmitting, isValid }) => {
           return (
-            <div className="flex flex-col md:w-full md:border md:p-8 md:rounded md:max-w-3xl md:mx-auto">
-              <BigHead
-                // @ts-ignore
-                className="border-b-4 h-36 md:h-48 border-primary-80"
-                eyebrows="raised"
-                faceMask={false}
-                lashes={false}
-                mask={false}
-                clothing={values.clothing}
-                hat="none"
-                graphic="none"
-                accessory={values.accessory}
-                body="chest"
-                skinTone={values.skinTone}
-                clothingColor={values.clothingColor}
-                eyes={values.eyes}
-                hair={values.hair}
-                hairColor={values.hairColor}
-                mouth={values.mouth}
-                facialHair={values.facialHair}
-              />
+            <Form>
+              <div className="flex flex-col md:w-full md:max-w-5xl md:mx-auto gap-4">
+                <div className="flex flex-col w-full md:flex-row gap-4">
+                  <div className="flex flex-col justify-between flex-1 order-2 gap-3 md:order-1 md:bg-slate-50 md:p-4 rounded">
+                    <div>
+                      <H3 className={{ root: 'border-b mb-0' }}>
+                        {t('shared.generic.profile')}
+                      </H3>
+                      <div className="space-y-3 mb-2">
+                        <FormikTextField
```

Comment:
new field for email, still planning to disable this if LTI login is used, so that email cannot be edited for LTI accounts.